docs/source/rfcs/lts.rst

```diff
@@ -0,0 +1,358 @@
+==========================
+ RFC: Long Term Stability
+==========================
+
+This RFC contains proposals and clarifications regarding the
+maintenance and release processes of Avocado.
+
+We understand there are multiple teams currently depending on the
+stability of Avocado and we don't want their work to be disrupted by
+incompatibilities nor instabilities in new releases.
+
+This version is a minor update to previous versions of the same RFC
+(see `Changelog`_) which drove the release of Avocado 36.0 LTS.  The
+Avocado team has plans for a new LTS release in the near future, so
+please consider reading and providing feedback on the proposals here.
+
+TL;DR
+=====
+
+We plan to keep the current approach of sprint releases every 3-4
+weeks, but we're introducing "Long Term Stability" releases which
+should be adopted in production environments where users can't keep up
+with frequent upgrades.
+
+Changelog
+=========
+
+Changes from `Version 3`_:
+
+ * Converted formatting to REStructuredText
+ * Replaced "me" mentions on version 1 changelog with proper name
+   (Ademar Reis)
+ * Renamed section "Misc Details" to `Deployment Details`_
+ * Renamed "avocado-vt" to "Avocado-VT"
+ * Start the timeline example with version 36.0
+ * Be explicit on timeline example that a minor bug did not generate
+   an immediate release
+
+Changes from `Version 2`_:
+
+ * Wording changes on second paragraph ("... nor instabilities...")
+ * Clarified on "Introduction" that change of behavior is introduced
+   between regular releases
+ * Updated distro versions for which official packages are built
+ * Add more clear explanation on official packages on the various
+   hardware platforms
+ * Used more recent version numbers as examples, and the planned
+   new LTS version too
+ * Explain how users can get the LTS version when using tools such as
+   pip
+ * Simplified the timeline example, with examples that will possibly
+   match the future versions and releases
+ * Documented current status of Avocado-VT releases and packages
+
+Changes from `Version 1`_:
+
+ * Changed "Support" to "Stability" and "supported" to "maintained"
+   [Jeff Nelson]
+ * Misc improvements and clarifications in the
+   supportability/stability statements [Jeff Nelson, Ademar Reis]
+ * Fixed a few typos [Jeff Nelson, Ademar Reis]
```

Comment:
Well, do you think this heading is necessary? I understand it in emails, where we propose different versions, but here I think it should start with the `Introduction`. For future updates we'd put those changes in commit message, the overall document should IMO be the final product.